app/models/brewery.rb

```diff
@@ -2,17 +2,16 @@
 
 class Brewery < ApplicationRecord
   # Elastic Search via Searchkick
-  searchkick word_start: %i[name city state]
+  searchkick
 
   geocoded_by :address
-  after_validation :geocode
 
   acts_as_mappable :lat_column_name => :latitude,
                    :lng_column_name => :longitude
 
   validates :name, presence: true
   validates :city, presence: true
-  validates :state, presence: true
+  validates :state, presence: true, if: :country_us?
```

Comment:
This is fine for now, but I eventually want to combine `state` and `county_province` into `state_region` or something similar. No action, just mentioning it for future thought :)

Reply:
So for US breweries it would be `dupage_illinois` and for international breweries (Ireland for example), it would be `laois`?

Reply:
Not exactly. Need to find a good standardization for addresses. This thread digs into the challenges (We want it to geocode but also searchable): 

https://stackoverflow.com/questions/3094126/best-practice-standard-for-storing-an-address-in-a-sql-database